### ISSUE TITLE
nuScenes Moving Object Segmentation Data

### ISSUE THREAD

**Issue by ZiliangMiao** [PRBonn/MapMOS#5]
Hi authors, thanks for your impressive job! Could you please provide the labeled nuScenes validation data, or can you explain how to label the dataset? This would be helpful for me to follow your work. Thanks and best regards.

**Comment by benemer** [comment]
Hey,

We use the attribute annotations which are provided by nuScenes. Please check our [dataloader](https://github.com/PRBonn/MapMOS/blob/main/src/mapmos/datasets/nuscenes.py) for more details.

Best
Benedikt

**Comment by benemer** [comment]
Note that you have to change the dataset version and split in this file to get the sequences from the validation set.

**Comment by ZiliangMiao** [comment]
Thanks for your reply!
